package.json

```diff
@@ -739,6 +738,13 @@
                 }
             ]
         },
+        "submenus": [
```

Comment:
Cloud9 probably doesn't support this, do we need a backlog item? Or is there an alternative way to reach this functionality for Cloud9?